docs/analytics/lineage/lineage.md

```diff
@@ -25,9 +25,9 @@ Provides details on the model classes & properties used within the query / servi
 ### Store Lineage
 
 Provides details on the `Store` features used in the query/mapping.  For example, if this is a
-relational / database `Store`, then this provides the link between the output columns and the 
-columns used from the specific database tables.  This can provide "dataset level lineage" or "attribute level lineage" 
-between the data model / service and the physical database. 
+relational / databaseType `Store`, then this provides the link between the output columns and the 
```

Comment:
Can you please revert these (typo?) changes of 'database' to 'databaseType' ? 

It will reduce the # of files being touched in this PR and in some cases 'database' is the right word to be used.

Reply:
right, my bad I have reverted those changes.